go/iot/main.go

```diff
@@ -93,22 +206,63 @@ func initOpenCensus(projectId string, reportPeriod int) {
 	}
 	view.RegisterExporter(exporter)
 
-	// Set reporting period to report data at every second.
+	// Set reporting period to report data based on the given reportPeriod.
 	view.SetReportingPeriod(time.Second * time.Duration(reportPeriod))
 
-	// Create view to see the processed video size cumulatively.
+	// Create view to see the sound strength distribution.
+	// Subscribe will allow view data to be exported.
+	// Once no longer need, you can unsubscribe from the view.
+	if err := view.Register(viewToRegister); err != nil {
+		log.Fatalf("Cannot subscribe to the view: %v", err)
+	}
+
+	// Create view to see the sound strength instantly.
+	// Subscribe will allow view data to be exported.
+	// Once no longer need, you can unsubscribe from the view.
+	if err := view.Register(&view.View{
```

Comment:
Same for this one. You can also make a slice of views that need registering, and register them in a loop.